client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/suggestion.js

```diff
@@ -12,7 +12,7 @@ import classNames from 'classnames';
  */
 import FormLabel from 'components/forms/form-label';
 import FormRadio from 'components/forms/form-radio';
-import ErrorNotice from 'components/error-notice';
+import Notice from 'components/notice';
```

Comment:
The `<ErrorNotice>` component had some CSS rules. Did you copied them to somewhere else?